### ISSUE TITLE
Interface hang and unnecessary focus on the text field when a new message appears in the chat

### ISSUE THREAD

**Issue by Dyaminigo** [UnigramDev/Unigram#1594]
I am using Unigram using the nvda screen reader. This is my main telegram client on windows and I really like it. But two problems are very annoying, the discussions of which I did not find in this repository:
1. When you read the message history in the chat / channel / group and at this moment a new message is added to the chat, the entire Unigram interface freezes and the screen reader does not respond to any key combination, and indeed no key combination works, even to close Unigram. Hangup automatically disappears after a few seconds. I don’t know what this may be connected with, but if the chat / channel / group is too active and new messages often appear there, then a lot of time is spent on waiting for the interface to freeze after each new message arrives.
2. If you read the history of messages in the chat / group in which you have the right to write new messages, and at this moment a new message is added, the screen reader will automatically focus on the text field. This is an incorrect behavior because you have to leave the text box each time. The focus on the list should be maintained when adding new messages to the end of the list, and not go to the text field. It is not comfortable.

I really hope that you can fix these two very annoying problems.
P.S. I apologize for my poor English.

**Comment by FrayxRulez** [comment]
Hello, to get rid of problem 1 you can disable In-App Sounds from Settings > Notifications and Sounds (this is actually a duplicate of #1503). I’m working on a boarder fix. 
On number 2 I would like to redirect you to #1557 as we’re discussing about it already 😊

**Comment by Dyaminigo** [comment]
@FrayxRulez
Oh! I should not have hoped so much for a keyword search. 😊
Thank you very much for pointing out the pages with a discussion of my concerns. I close this issue, I will follow the previously created issues.
